src/mon/ConfigMonitor.cc

```diff
@@ -141,6 +141,21 @@ static string indent_who(const string& who)
   return "    " + who;
 }
 
+static void print_value(Formatter *f, bufferlist &odata,
+                        const string &name, const string &value)
+{
+  if (f) {
+    f->open_object_section("config");
+    f->dump_string("name", name);
+    f->dump_string("value", value);
```

Comment:
as I noted for `config show`, we should reuse `print_value` for the vanilla `config get <entity>` and include any fields it has missing here.